File: IntroToLINQ/IntroToLINQ_CSharp/MyDbContext.cs
using System.Data.Entity;

namespace IntroToLINQ
{
    public class MyDbContext : DbContext
    {
        public MyDbContext(string name) : base(name)
        {
            Database.SetInitializer(new DropCreateDatabaseIfModelChanges<MyDbContext>());
        }
        public DbSet<Student> Students { get; set; }
    }
}

File: IntroToLINQ/IntroToLINQ_VB/MyDbContext.vb
Imports System.Data.Entity
Imports ConsoleApp15

Public Class MyDbContext
    Inherits DbContext

    Public Sub New(ByVal name As String)
        MyBase.New(name)
        Database.SetInitializer(New MyDbInitialiser)
    End Sub

    Public Property Students As DbSet(Of Student)
End Class

Public Class MyDbInitialiser
    Inherits DropCreateDatabaseIfModelChanges(Of MyDbContext)

    Protected Overrides Sub Seed(context As MyDbContext)
        For Each st As Student In CreateStudents()
            context.Students.Add(st)
        Next
        context.SaveChanges()
    End Sub

End Class